### DISCUSSION TITLE
NEP training not running on multiple GPU nodes

### DISCUSSION THREAD

**Discussion by Sonti974948** [brucefan1983/GPUMD#731] [Q&A] [answered]
Hi GPUMD team, 

I was trying to obtain a scaling plot to compare the speed of training with increasing number of GPUs, and though I was able to obtain a good scaling within 1 GPU node (i.e. across 1,2,4 GPUs), when I submitted a job with 2 GPU nodes (i.e. 8 GPUs), the training was still using only 1 GPU node (4 GPUs). 

Is there a way to train a NEP model on 2 different nodes?

Thank you so much!
Siddharth Sonti

**Comment by brucefan1983** [comment, marked as answer]
Thanks for the question. There is curretly only support for single-node parallelization. To extend to more nodes, MPI is needed, but we have not implemented MPI yet.

**Reply by Sonti974948** [reply]
Thank you for the clarification!
